src/aws-cpp-sdk-core/include/aws/core/VersionConfig.h.in

```diff
@@ -3,7 +3,15 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 
+// DEPRECATED: This file is used by legacy_build generators only
+
 #define AWS_SDK_VERSION_STRING "@AWSSDK_VERSION_STRING@"
 #define AWS_SDK_VERSION_MAJOR @AWSSDK_VERSION_MAJOR@
 #define AWS_SDK_VERSION_MINOR @AWSSDK_VERSION_MINOR@
-#define AWS_SDK_VERSION_PATCH @AWSSDK_VERSION_PATCH@
\ No newline at end of file
+#define AWS_SDK_VERSION_PATCH @AWSSDK_VERSION_PATCH@
+// Forward compatible definitions
+#define AWS_CPP_SDK_VERSION_STRING AWS_SDK_VERSION_STRING
+#define AWS_CPP_SDK_VERSION_MAJOR AWS_SDK_VERSION_MAJOR
+#define AWS_CPP_SDK_VERSION_MINOR AWS_SDK_VERSION_MINOR
+#define AWS_CPP_SDK_VERSION_PATCH AWS_SDK_VERSION_PATCH
+#define AWS_CPP_SDK_GIT_HASH "NoHashAvailableInLegacyBuild"
```

Comment:
Added for forward compatibilty

Reply:
nit: I know CPP SDK sounds better, but we have AWS SDK CPP in most of the places.
nit 2: maybe GIT_SHA1 or _REVISION or GIT_TAG?